app/models/category.rb

```diff
@@ -1,3 +1,13 @@
 class Category < ApplicationRecord
+    after_initialize :set_default_status
+    
     has_many :articles
+    validates_presence_of :name
+    validates_presence_of :status
```

Comment:
Hey - really sorry but I think something seems strange here - I'm not sure we need 2 attributes for category, only title (not status). What values fit into the `:name` attribute in this setup vs `:status`? I'm wondering if this is because it was difficult to define `:name`?